pkgs/development/python-modules/impit/default.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  rustPlatform,
+  cargo,
+  rustc,
+  perl, # for openssl
+  pytestCheckHook,
+}:
+buildPythonPackage rec {
+  pname = "impit";
+  version = "0.7.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "apify";
+    repo = pname;
+    rev = "py-${version}";
+    hash = "sha256-2PNxobTMFaxJCPKb8YFR576r3uzWpymDw6UF5xGj2OA=";
+  };
+
+  buildAndTestSubdir = "impit-python";
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit src;
+    hash = "sha256-LH3cDn16VWSNk8xYk7l5AvnmoMk3dgdi+LSjq6vcAns=";
+  };
+
+  RUSTFLAGS = "--cfg reqwest_unstable";
```

```suggestion
  env.RUSTFLAGS = "--cfg reqwest_unstable";
```
Add a comment with the reason.